SETUP.md

```diff
@@ -1,30 +1,30 @@
 # Welcome!
 
-It's quick and easy to set up your environment to build [New Relic One applications](https://docs.newrelic.com/docs/new-relic-programmable-platform-introduction). In these setup instructions, you'll be setting up your system to run a local server that establishes a connection to the New Relic One platform. This connection makes building your New Relic One application easy because you can see your changes in real time. Enjoy!
+You can set up your environment to build [New Relic One applications](https://docs.newrelic.com/docs/new-relic-programmable-platform-introduction) in three steps. By the end of this guide, you'll have a local server with an established connection to the New Relic One platform, where you can see changes to your app in real time. Enjoy!
 
-# Setup Instructions
+## Setting up your environment
 
-To set up your environment to Build your own application:
+1. In your browser, go to [New Relic One](https://one.newrelic.com/) and follow the instructions to [**Build your own application**](https://one.newrelic.com/launcher/developer-center.launcher#pane=eyJuZXJkbGV0SWQiOiJkZXZlbG9wZXItY2VudGVyLmRldmVsb3Blci1jZW50ZXIifQ==).
 
-1. In Google Chrome, go to [one.newrelic.com](https://one.newrelic.com/) and select [**build your own application**](https://one.newrelic.com/launcher/developer-center.launcher#pane=eyJuZXJkbGV0SWQiOiJkZXZlbG9wZXItY2VudGVyLmRldmVsb3Blci1jZW50ZXIifQ==). Follow the instructions.
```

Comment:
Don't specify Google Chrome, as any browser will do. Also, use clear words to describe where the hyperlink will take the user.